src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/command/rulemanager/OfBatchExecutor.java

```diff
@@ -62,8 +61,8 @@ public class OfBatchExecutor {
     private final OfBatchHolder holder;
     private final Set<SwitchFeature> switchFeatures;
     private final String kafkaKey;
-    private final Origin origin;
     private final boolean failIfExists;
+    private final String sender;
```

Comment:
It's better to name it something like `incomeTopic` or `sourceTopic`. Be more accurate with naming.